tools/specs/add-ons/Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.nuspec

```diff
@@ -24,7 +24,7 @@ Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyStoreProvider.SqlColumnEncrypti
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <tags>sqlclient microsoft.data.sqlclient azurekeyvaultprovider akvprovider alwaysencrypted</tags>
     <dependencies>
-      <group targetFramework="net46">
+      <group targetFramework="net461">
```

Comment:
Let's not touch AKV provider project right now.
This PR shall only target changes in NetFx driver.

Reply:
done.